kube-client/src/api/mod.rs

```diff
@@ -15,6 +15,11 @@ mod subresource;
 pub use subresource::{Attach, AttachParams, Execute, Portforward};
 pub use subresource::{Evict, EvictParams, Log, LogParams, ScaleSpec, ScaleStatus};
 
+// Ephemeral containers were stabilized in Kubernetes 1.25.
+k8s_openapi::k8s_if_ge_1_25! {
```

Comment:
hmm, you would at the very least need the same gate the integration tests in lib.rs as well because they'll not pass the MK8SV check without it. but otherwise it's a pretty clean cutoff.